src/main/java/org/osiam/resources/controller/ServiceProviderConfigsController.java

```diff
@@ -50,7 +48,7 @@ public ServiceProviderConfig getConfig() {
         public static final String SCHEMA = "urn:ietf:params:scim:schemas:core:2.0:ServiceProviderConfig";
         public final Supported patch = new Supported(true);
         public final Supported bulk = new BulkSupported(false);
-        public final Supported filter = new FilterSupported(true, RequestParamHelper.MAX_RESULT);
+        public final Supported filter = new FilterSupported(true, 100);
```

Comment:
I think the constant does belong in the `ScimSearchResult` and already spoke with @dacrome about it. I will provide a follow-up PR after this one is merged.